app/models/category.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require 'common/models/base'
+
+# Category model
+class Category < Common::Base
+  def category_id
+    0
```

Comment:
Yes - category-id is always a 0. Its really kind of a pseudo-resource that is always the same, but may change at some point. Didn't want the FE to have to hard-code it.